content/en/account_management/org_settings/ip_allowlist.md

```diff
@@ -3,7 +3,7 @@ title: IP Allowlist
 ---
 
 {{< callout url="/help/" header="Get Started with IP Allowlist" >}}
-The IP allowlist feature is available for customers on an enterprise plan only. Request access by contacting support.
+The IP allowlist feature is exclusively available to customers on the pro+ and enterprise plans. To request access, please contact support.
```

Comment:
Recommend adding a link to support: https://help.datadoghq.com/hc/en-us

```suggestion
The IP allowlist feature is exclusively available to customers on the Pro+ and Enterprise plans. To request access, contact support.
```

Reply:
Hi @estherk15, 

Thanks for the comment! In the button bellow we're using the following link: https://docs.datadoghq.com/help/

<img width="750" alt="image" src="https://github.com/user-attachments/assets/f864d76b-5cfa-4eb4-9030-9c32f6fbc384">

Do you think we should: 
1. Replace the link in the button by the one you shared, 
2. Add both links? 

Having both links might be confusing.

What do you think?

Reply:
@anthony-dgx My mistake, I didn't see the URL, I think this is good as is!